doc/dev/development.rst

```diff
@@ -404,36 +404,45 @@ tagging process is as follows:
    core development team will decide to merge the ``develop`` branch into
    ``release``.
 
- * A branch is created off of ``develop`` (typically called ``staged``)
-   and then a `PR <https://github.com/pypeit/PypeIt/compare>`_ is issued
-   to merge ``staged`` into ``release``.  This merge must meet the same
-   `Pull Request Acceptance Requirements`_ when merging new branches
-   into ``develop``.  However, typically the tests do not need to be run because
-   ``develop`` has already passed these tests and ``staged`` will not make any
-   substantive changes to the code.
-
- * **Before being merged into release**, the code is tagged as follows:
-
+ * A branch is created off of ``develop`` (typically called ``staged``) and then
+   a `PR <https://github.com/pypeit/PypeIt/compare>`_ is issued to merge
+   ``staged`` into ``release``.  This ``release...staged`` PR must meet the same
+   `Pull Request Acceptance Requirements`_ when merging new branches into
+   ``develop``.  Code review is expected to be limited (because all code changes
+   will have been reviewed before pulling into ``develop``), but the result of
+   the dev-suite tests must be shown and approved.  The reason for creating the
+   new branch instead of a direct ``release...develop`` PR is to allow for the
+   following updates to ``develop`` before merging (``develop`` is a protected
```

Comment:
Did you mean to say:
```
following updates to ``staged`` before merging (``develop`` is a protected
```
?

Reply:
Yeah, good catch.  I see how that's confusing.  Changed.